src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -40,33 +42,37 @@ public LineResponse saveLine(LineRequest request) {
     public List<LineResponse> findLines() {
         List<Line> persistLines = lineRepository.findAll();
         return persistLines.stream()
-                .map(line -> {
-                    List<StationResponse> stations = line.getStations()
-                            .stream()
-                            .map(it -> StationResponse.of(it))
-                            .collect(Collectors.toList());
-                    return LineResponse.of(line, stations);
-                })
-                .collect(Collectors.toList());
+            .map(line -> {
+                List<StationResponse> stations = line.getStations()
+                    .stream()
+                    .map(it -> StationResponse.of(it))
+                    .collect(Collectors.toList());
+                return LineResponse.of(line, stations);
+            })
+            .collect(Collectors.toList());
+    }
+
+    public List<Line> findAllLines() {
+        return lineRepository.findAll();
     }
 
-    public Line findLineById(Long id) {
-        return lineRepository.findById(id).orElseThrow(RuntimeException::new);
+    public Line findById(Long id) {
+        return lineRepository.findById(id)
+            .orElseThrow(() -> new IllegalArgumentException("노선을 찾을수 없습니다"));
     }
 
 
     public LineResponse findLineResponseById(Long id) {
-        Line persistLine = findLineById(id);
+        Line persistLine = findById(id);
         List<StationResponse> stations = persistLine.getStations()
-                .stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
+            .stream()
+            .map(it -> StationResponse.of(it))
```

Comment:
메서드 참조도 활용해 볼 수 있겠네요
```suggestion
            .map(StationResponse::of)
```

Reply:
감사합니다! 놓친부분이었네요.